Package.swift

```diff
@@ -4,15 +4,15 @@
 import PackageDescription
 
 let package = Package(
-    name: "JellyfishClientSdk",
+    name: "FishjamClientSdk",
     platforms: [
         .iOS(.v13)
     ],
     products: [
         // Products define the executables and libraries a package produces, and make them visible to other packages.
         .library(
-            name: "JellyfishClientSdk",
-            targets: ["JellyfishClientSdk"])
+            name: "FishjamClientSdk",
```

Comment:
We should stick with the same name across package managers. On cocapods we have `FishjamClient` it should be named the same here.

Reply:
I updated it everywhere to match the naming.